asyncapi/twitter/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Cost/Freemium" , "Vendor/Twitter"]
+org = "ballerinax"
+name = "trigger.twitter"
+repository = "https://github.com/ballerina-platform/asyncapi-triggers"
+version = "0.1.0"
```

Comment:
Let's go with a major release 1.0.0

Reply:
Ack. Version is updated as major release